From b8ca7606ac83ae6979ee3772f6268a2a62c1fb8b Mon Sep 17 00:00:00 2001
From: Nicolin Chen <B42378@freescale.com>
Date: Sun, 28 Apr 2013 17:38:35 +0800
Subject: [PATCH 391/435] ENGR00260754 mxc: asrc: fix memory leak

Added free() for 'input_null' which's allocated but wasn't correctly freed.

Signed-off-by: Nicolin Chen <b42378@freescale.com>o
---
 test/mxc_asrc_test/mxc_asrc_test.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/mxc_asrc_test/mxc_asrc_test.c b/test/mxc_asrc_test/mxc_asrc_test.c
index 2165ca0..8e3145a 100644
--- a/test/mxc_asrc_test/mxc_asrc_test.c
+++ b/test/mxc_asrc_test/mxc_asrc_test.c
@@ -608,6 +608,7 @@ int main(int ac, char *av[])
 	fclose(fd_dst);
 	close(fd_asrc);
 
+	free(input_null);
 	free(input_buffer);
 	free(output_buffer);
 	printf("All tests passed with success\n");
-- 
1.8.0

